benchmark/scala/src/main/scala/com/microsoft/tpch/ConvertTpchCsvToParquetApp.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+package mypackage
+
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{SaveMode, SparkSession}
+
+object ConvertTpchCsvToParquetApp {
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession.builder()
+      .appName(this.getClass.getName)
+      .getOrCreate()
+
+    // The 1st arg should be the folder containing TPCH tables.
+    val tpchSourceFolderPath = args(1)
```

Comment:
I thought it is 0 based, no?

Reply:
I just tested this, it is zero based. My bad, I was looking at the wrong example. Thanks.